Jenkinsfile

```diff
@@ -1,10 +1,10 @@
-@Library('github.com/fabric8io/fabric8-pipeline-library@master')
+@Library('github.com/hectorj2f/fabric8-pipeline-library@dockerbaseimage9')
```

Comment:
this needs to be changed to the upstream repo once this is merged into the pipeline library